src/assets/icon-private.svg

```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 40 40">
```

Comment:
This file can be minified by almost 50%. Can you try running it through any cloud minifier like www.svgminify.com ?

Reply:
Done